node/tests/sync_test.go

```diff
@@ -37,17 +36,79 @@ func TestSyncLightWithBridge(t *testing.T) {
 
 	err := bridge.Start(ctx)
 	require.NoError(t, err)
-	time.Sleep(1 * time.Second)
-	assert.False(t, bridge.HeaderServ.IsSyncing())
+	h, err := bridge.HeaderServ.GetByHeight(ctx, 20)
+	require.NoError(t, err)
```

Comment:
Pls make the context here is wrapped with a timeout. This is important to follow in every test

Reply:
I've made the ctxTimeout for GetByHeight only.
wdyt?

Reply:
I am ok with this as long as there is a global context with a timeout. The global context is needed so we don't ever deadlock while running tests locally or on CI.

Reply:
The blocking may happen literally everywhere where context is passed and not only for GetByHeight, so the timeout should be on the root context. Further, it won't really make sense to have a special context timeout for GetByHeight as there will be one in the root.

Reply:
Ok, I've changed to global ctx with timeout, so resolving and merging 
thanks for input 🙏